SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -219,6 +219,24 @@ public class SwiftyTesseract {
     
     return data
   }
+
+  public func recognizedBlocksByLevel(_ level: ResultIteratorLevel) -> [RecognizedBlock] {
+
+    let tess_level = level.asTessarctLevel
```

Comment:
While there is no official project style guide, we don't use any snake case here. I get why you would maybe prefer to do this since it's a C type, however the rest of the codebase doesn't follow that convention so this and all other snake case variable declarations should be converted to camel case.